agent/custom/general_navigation_action.py

```
@@ -1,238 +1,482 @@
+                last_set = cur_set
+
+                # 5d. 向下滑动(手指上滑, 看下方地图) 200px, 移动后停顿1S再检测
+                controller.post_swipe(640, 600, 640, 600 - SWIPE_DIST, SWIPE_DURATION).wait()
+                swipe_count += 1
+                time.sleep(1.0)
+
+            mfaalog.error(f"[导航] 滑到底未找到目标关卡: {target_quest} (共滑动 {swipe_count} 次)")
             return CustomAction.RunResult(success=False)
-                
+
         except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)